src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -636,6 +636,19 @@ public RequiredMemberAttribute()
         }
     }
 }
+";
+
+        protected const string SetsRequiredMembersAttribute = @"
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Constructor, Inherited = false, AllowMultiple = false)]
+    public class SetsRequiredMembersAttribute : Attribute
```

Comment:
nit: would there be a point in adding 'sealed' here and in 'class RequiredMemberAttribute' like we have in 'UnmanagedCallersOnly', etc. above?

Reply:
Probably. I'll do that in the next PR.